src/component/organism/info/SocialInfoB.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import {Text, View} from 'react-native';
 import {txt} from 'Root/config/textstyle';
-import {socialInfoB} from '../style_organism';
+import {socialInfoB} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {socialInfoB} from 'Organism/style_organism copy';
```